swatch-metrics/deploy/clowdapp.yaml

```
@@ -113,8 +117,8 @@ objects:
         - swatch-tally
 
       kafkaTopics:
-        - replicas: 3
-          partitions: 3
+        - replicas: ${{KAFKA_METERING_TASKS_REPLICAS}}
+        partitions: ${{KAFKA_METERING_TASKS_PARTITIONS}}
```

Looks like it fails to deploy due to accidental mis-indentation:

```suggestion
        - replicas: ${{KAFKA_METERING_TASKS_REPLICAS}}
          partitions: ${{KAFKA_METERING_TASKS_PARTITIONS}}
```